packages/deprecated/http/httpcall_server.js

```diff
@@ -162,4 +162,10 @@ function _call (method, url, options, callback) {
 }
 
 // we are keeping wrapAsync here as this package is deprecated
```

Comment:
This comment is out of date.

Also, I wanted to note that the change to HTTP.call is a breaking change, so we will need to bump the major version for the package.